backend/backend/properties.py

```diff
@@ -254,3 +269,34 @@ def _is_in_monopoly(self):
         if result[0] == 1:
             is_monopoly = True
         return is_monopoly
+
+    def rails_owned(self):
+        """
+        Returns:
+            int: how many railroads are owned by the owner of self.
+        """
+        with self._conn.cursor() as cursor:
+            cursor.execute('SELECT COUNT(*) FROM `properties` ',
+                           'WHERE `game_id` = %s ',
+                           'AND `property_type` = `railroad` ',
+                           'AND `player_id` = %s',
```

Comment:
In regards to line 279 to 282, from looking at the db tables you need to make a join query with _properties_ table to get the player_id and with _property_values_ table to get the _state_ (the property_type) of the property to sum the number of _railroad_ type property's owned. _property_type_ isn't a column in the properties table.